service/src/test/java/bio/terra/externalcreds/controllers/SshKeyApiControllerTest.java

```diff
@@ -108,18 +145,11 @@ void samThrowsApiException() throws Exception {
     when(usersApiMock.getUserStatusInfo()).thenThrow(new ApiException());
 
     var sshKeyPairType = "gitlab";
-    var rsaEncodedKeyPair = TestUtils.getRSAEncodedKeyPair(externalUserEmail);
-    var sshKeyPair =
-        new SshKeyPair()
-            .privateKey(rsaEncodedKeyPair.getLeft())
-            .publicKey(rsaEncodedKeyPair.getRight())
-            .externalUserEmail(externalUserEmail);
-    var requestBody = objectMapper.writeValueAsString(sshKeyPair);
     mvc.perform(
-            put("/api/sshkeypair/v1/{type}", sshKeyPairType)
+            post("/api/sshkeypair/v1/{type}", sshKeyPairType)
```

Comment:
why move these tests from the 'set' to the 'generate'?  do we want them on both, or just leave this on 'set'?

Reply:
shouldn't really matter which method we are calling because it's the same endpoint. I actually like using the post one then put as there's less set up.